github/transport.go

```diff
@@ -13,8 +13,9 @@ import (
 )
 
 const (
-	ctxEtag = ctxEtagType("etag")
-	ctxId   = ctxIdType("id")
+	ctxEtag        = ctxEtagType("etag")
+	ctxId          = ctxIdType("id")
+	packageVersion = "4.26.1"
```

Comment:
Someone will have to update this when the version is bumped - perhaps there is a way to avoid that!